semantic-analysis/.ghci.repl

```diff
@@ -0,0 +1,40 @@
+-- GHCI settings, collected by running cabal repl -v and checking out the flags cabal passes to ghc.
```

Comment:
This and several other files exist solely to make local development of just `semantic-analysis` more convenient.